src/main/java/de/dennisguse/opentracks/TrackRecordingActivity.java

```diff
@@ -269,6 +274,14 @@ public boolean onOptionsItemSelected(MenuItem item) {
             return true;
         }
 
+        if (item.getItemId() == R.id.track_detail_insert_marker) {
```

Comment:
should still be the first entry in the menu

Reply:
The first one was Map.
What would be your favorite order?

I've decided this:

1. Profiles
2. Map
3. Insert Marker

because I use the more profiles to change between running and cycling, then I usually use the map and I seldom add markers.

Anyway, I don't care about this. Because of this, let me know what the order should be and I'll do it ;)

Reply:
IMHO: map, insert marker, profiles.

(I often open the map and then close it again)